ASWF/meetings/tsc/2020-09-28.md

```diff
@@ -0,0 +1,222 @@
+  * [ ] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Bernard Lefebvre - Autodesk
+  * [X] Nigel Hadley - Foundry
+  * [X] Simran Spiller
+
+Apologies:
+  * Kevin Wheatley
+  * Mark Titchener
+
+# **OCIO TSC Meeting Notes**
+
+* TAC update on GSoC. Focus for next year:
+    - Michael: Good feedback on results. Likely that ASWF will participate 
+      again next year and are hopful that all projects can get a project. 
+      Should start thinking about where to focus. CUDA, Metal renderer impl 
+      seems like a good place to try again on.
+    - Doug: Good to have volunteer for each project. Is Hacktoberfest 
+      happening again this year?
+    - Michael: Good question. I believe that was based on a tag search.
+
+* Issue #1117 -- Help Wanted:
+    - Doug: Added new label help-wanted. Others use this too. Can start adding 
+      this to high priority issues. This issue is to create OFX bindings for 
+      OCIO v2. High priority since it would increase test coverage with so many 
+      supporting apps. Could get more practitioners and developers to download 
+      and test those.
+    - Michael: OFX is supported by Nuke; Curious if it would create conflict 
+      with existing OCIO v1 binary in that env.
+
+* Open PRs: 
+    - 1147 (optimization):
+        - Doug: Updates config default version and optimization improvements. 
+          When create new config, it is now v2 by default instead of v1. Other 
+          thing in PR enhances optimization flags. Now ability to control types 
+          of optimization. Two new flags: one simplifies ops down to component 
+          parts; CDL without power becomes just matrix if possible, etc. 
+          Benefit is that can be combined with another matrix op. Another flag 
+          controls whether SSE log and power are used. Does much faster version 
+          of these functions but not 32-bit exact - 16-bit half precision. 
+          Useful to turn it on and off, and off for unit tests since we’re 
+          comparing with higher precision results.
+        - Bernard: Fast log is also used for baking.
+        - Mark: Why is SSE path not as precise? Because of writing own exponent 
+          functions etc?
+        - Patrick: Yes, wrote own implementation. Benefit in performance is so 
+          important that both approaches are supported.
+        - Doug: Uses fewer iterations than needed for full float 32-bit results.
+        - Mark: Had similar challenge getting this working in CTL expression 
+          code. Finding concise method of operations challenging. Since we 
+          already have concept of optimizations sounds fine. Is this default?
+        - Doug: Yes, both are turned on by default, in OPTIMIZATION_VERY_GOOD.
+        - Patrick: optimization is based on bits, so can disable any of the 
+          items. Fine control of optimization. There was a question from Zach 
+          Lewis about v1. You can still make v1 config by setting version.
```

Comment:
👍
Regarding optimization flags and versions, I was curious to know if you guys had given any consideration to format / version compatibility optimization flags / filters...? I.e., something like an OPTIMIZATION_COMPAT_OCIO_1_1 option that would constrain an optimized processor to a subset of ostensibly backward-compatible transforms; or OPTIMIZATION_COMPAT_CTF_1_3 for Flame-2019(?)+ compatible-transforms (if that's even possible); and so on, for future versions of CTF / CLF / OCIO / etc.

(Not a feature request, just curious).

Reply:
We have not implemented that.